nri-kafka/src/Kafka/Worker.hs

```diff
@@ -26,19 +26,9 @@ module Kafka.Worker
     Internal.subscriptionManageOwnOffsets,
     Internal.Envelope (topicName, partitionId, messageOffset, payload),
     Partition.SeekCmd (..),
-
-    -- * Wrapper types
-    Internal.OffsetSource (..),
-    Consumer.ConsumerGroupId (..),
```

Comment:
I believe we still need to export this now that it's an argument

Reply:
I think we should consider taking a `Text` argument and wrapping it ourselves.